kokoro/gcp_ubuntu/kokoro_build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Fail on any error.
+set -e
+
+cd "${KOKORO_ARTIFACTS_DIR}/github/raksha"
```

Comment:
I vaguely remember having to setup a docker container somewhere to have a predictable environment. Where is that done?

Reply:
It did not appear in the Kokoro codelab for Ubuntu on GCP. I don't know whether they simplified it or if it's more advanced configuration that we'll have to iterate on later.